packages/compass-e2e-tests/tests/assistant.test.ts

```diff
@@ -175,6 +175,13 @@ describe('MongoDB Assistant', function () {
       );
       throw err;
     }
+
+    // Close the drawer if open to provide a clean environment for the next test
+    const drawerCloseButton = browser.$(Selectors.AssistantDrawerCloseButton);
+    if (await drawerCloseButton.isDisplayed()) {
+      await browser.clickVisible(drawerCloseButton);
+      await drawerCloseButton.waitForDisplayed({ reverse: true });
+    }
```

Comment:
I'm suggesting closing the drawer (if open) to provide a clean environment between tests.